packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -191,6 +131,7 @@ const setDidToNotRotatedState = (did: DID) => {
   did.createJWS = async () => jwsForVersion0
 }
 
+// TODO: De-dupe this with similar code from tile-document-handler.test.ts and model-instance-document.test.ts
```

Comment:
I'm pretty disappointed about this.  There is quite a lot of somewhat nasty test code here for managing key rotations that is completely duplicated between this test and the equivalent tests for TileDocumentHandler and ModelInstanceDocumentHandler.  I tried to pull this code into `TestUtils`, but unfortunately moving this into the `common` package caused a bunch of circular dependencies that broke compilation.  I think sometime soon we're going to need to rethink our package structure and create some more shared packages for helper functionality and test functionality that can be shared across the majority of our packages.  But for now it was simplest just to leave this code duplicated.  I don't love it, but at least it's only test code.